### DISCUSSION TITLE
GRBL Firmware Update Feature -  Needs some clarification.

### DISCUSSION THREAD

**Discussion by EricAiken** [arkypita/LaserGRBL#2143] [Q&A]
I'm trying to update the firmware on my OLM2.  

Within LaserGRBL there is an option to flash GRBL firmware (not olm2), but it has a caveat.

Warning! this operation can damage your hardware.  The operation can be irreversible: use at your risk.
This procedure is not intended for Ortur Engraver.!

That last line is quite clear.

My OLM2 is running 1.1f (2017) which is quite old.  There is a newer 1.1h, but this warning says don't.  Can someone enlighten me on the purpose of this GRBL firmware update.  Why are the ortur's excluded?  Is just a a specific model of ortur or all of them?

Thanks in advance

**Comment by arkypita** [comment]
Ortur use a chip (STM32) that is not the atmega328 for which the procedure is designed.
You need to download their own firmware, that is designed for STM32, and follow their flash instruction.

**Comment by wortex1979** [comment]
′

22 Eki 2023 Paz 09:01 tarihinde arkypita ***@***.***> şunu
yazdı:
